burr/core/application.py

```diff
@@ -395,11 +397,11 @@ def _process_inputs(self, inputs: Dict[str, Any], action: Action) -> Dict[str, A
         for key in list(inputs.keys()):
             if key in action.inputs:
                 processed_inputs[key] = inputs.pop(key)
-        if len(inputs) > 0:
-            raise ValueError(
+        if len(inputs) > 0 and logger.isEnabledFor(logging.DEBUG):
+            logger.debug(
                 f"Keys {inputs.keys()} were passed in as inputs to action "
-                f"{action.name}, but not declared by the action as an input! "
-                f"Action needs: {action.inputs}"
+                f"{action.name}, but not declared by the action as an input. "
+                f"Action only needs: {action.inputs}."
```

Comment:
Reword this, with the new functionality its supposed to happen. Even in debug it'll confuse people, so add more clarification.